### PR TITLE
Translated to Czech

### PR DESCRIPTION
**Pull request by VojtechTM**
i translated the accesibility slide to czech

### CODE REVIEW

**Comment by whyadamsalt** [comment]
Would it be fine if you translated all of the others too? It would be easier if all the slides were translated otherwise contributors may not realize that only one of them have been done.

**Comment by VojtechTM** [comment]
Yeah, im planning to do that, but i have only 15 mins of internet so i just pushed this

**Comment by whyadamsalt** [comment]
That's fine, is it alright if we merge once all are translated?

**Comment by VojtechTM** [comment]
Yeah, of course!

**Comment by VojtechTM** [comment]
Managed to translate 3 pages today, gonna do the rest tommorow

**Comment by whyadamsalt** [comment]
Okay that's brilliant, thank you!

**Comment by whyadamsalt** [comment]
Are all of the pages done?

**Comment by ghost** [comment]
I want to follow up, are all of the pages done for the translation?

**Comment by VojtechTM** [comment]
Sorry i've been really busy, im gonna try to finish it up by saturday

**Comment by VojtechTM** [comment]
You can know merge it into mainstream!

File: Assets/ubiquity-slideshow/slides/cs/gethelp.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<div class="main wide">
+
+<div class="text">
+
+<div>
+<p>Protože CelOS je založeno na Ubuntu, pomoc najdete v dokumentaci pro Ubuntu</p>
+</div>
+
+<div>
+<p>At <a href="http://askubuntu.com">Ask Ubuntu</a> Můžete se zeptat na otázky a hledat ve velké kolekci už odpovězených otázek</p>
```
Comment (moeenio) [inline Assets/ubiquity-slideshow/slides/cs/gethelp.html]: Is the "At" translated here?

File: Assets/ubiquity-slideshow/slides/cs/gethelp.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+</div>
+
+<div>
+<p>Pro další užitečné informace jděte na <a href="https://www.ubuntu.com/support/community-support">Ubuntu komunitní podopra</a> or <a href="http://www.ubuntu.com/support">Ubuntu komerční podpora</a>.</p>
```
Comment (moeenio) [inline Assets/ubiquity-slideshow/slides/cs/gethelp.html]: Same for the "or"?

File: Assets/ubiquity-slideshow/slides/cs/photos.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+</ul>
+<h2 class="subtitle">Doporučené programy</h2>
+<ul>
+	<li>
+	<img class="icon" src="icons/gimp.png" />
+	<p class="caption">GIMP editor obrázků</p>
+	</li>
+	<li>
+	<img class="icon" src="icons/shotcut.png" />
+	<p class="caption">Shotcut Video Editor</p>
```
Comment (moeenio) [inline Assets/ubiquity-slideshow/slides/cs/photos.html:27]: Forgot to translate?

**Review by moeenio — CHANGES_REQUESTED** [review]
I don't speak any Czech but I feel like there are some typos

File: Assets/ubiquity-slideshow/slides/cs/gethelp.html (hunk 1 above)
Comment (VojtechTM) [reply Assets/ubiquity-slideshow/slides/cs/gethelp.html]: sorry, didnt notice that

File: Assets/ubiquity-slideshow/slides/cs/photos.html (hunk 3 above)
Comment (VojtechTM) [reply Assets/ubiquity-slideshow/slides/cs/photos.html:27]: Thats the same

Comment (moeenio) [reply Assets/ubiquity-slideshow/slides/cs/photos.html:27]: I mean, it looks like you've translated the names of all other programs. Why would you leave this one intact?

**Comment by VojtechTM** [comment]
fixed the typos.

File: Assets/ubiquity-slideshow/slides/cs/photos.html (hunk 3 above)
Comment (VojtechTM) [reply Assets/ubiquity-slideshow/slides/cs/photos.html:27]: because video editor is the same in czech

**Review by moeenio — APPROVED** [review]

**Comment by VojtechTM** [comment]
Was the translation merged? cant seem to find it

**Comment by moeenio** [comment]
It wasn't, as you can see the PR is still open. I've been working on improving the folder structure, so I'll have to update this pr after that.

**Comment by VojtechTM** [comment]
ok, let me know if you need any help

**Comment by moeenio** [comment]
Any help on #48 would be very appreciated.

**Comment by VojtechTM** [comment]
I've had shortend the readme file, and created a docs folder where ive put the packages

**Comment by moeenio** [comment]
Please create another PR for the beta7 branch.

**Comment by moeenio** [comment]
Closing this for now. Translations will be considered later.

**Pull request closed without merging**